node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Template.java

```diff
@@ -50,7 +49,7 @@ public FileWriter getFileWriterTo(Path destinationPath) {
 
     public String render() {
         StringWriter writer = new StringWriter();
-        velocityEngine.evaluate(velocityContext, writer, "Template", template);
+        Velocity.evaluate(velocityContext, writer, "Template", template);
```

Comment:
This appears to use synchronized around a singleton, but we're not that multi-threaded that it would cause a problem.

Reply:
Nope.